### PR TITLE
Update graceful shutdown docs

### PR DESCRIPTION
**Pull request by adleong**

### CODE REVIEW

**Review by kflynn — APPROVED** [review]
This looks good to me, though I think @alpeb's review before merging would be good too. 🙂

File: linkerd.io/content/2.16/tasks/graceful-shutdown.md [hunk 1]
```diff
@@ -134,9 +134,20 @@ containers in the pod complete. However, the Linkerd proxy container runs
...
 proxy a means to know when the Cronjob has completed, by default, Job and
 Cronjob pods which have been meshed will continue to run even once the main
-container has completed.
+container has completed. You can address this either by running Linkerd as a
+native sidecar or by manually shutting down the proxy.
 
-To address this, you can issue a POST to the `/shutdown` endpoint on the proxy
+### Native Sidecar
+
+If you use the `--set nativeSidecar=true` flag when installing Linkerd, the
```
Comment (alpeb) [inline linkerd.io/content/2.16/tasks/graceful-shutdown.md:142]: Should be `--set proxy.nativeSidecar=true`

File: linkerd.io/content/2.16/tasks/graceful-shutdown.md [hunk 2]
```diff
@@ -147,5 +158,7 @@ One convenient way to call this endpoint is to wrap your application with the
 application that is called this way (e.g. via `linkerd-await -S $MYAPP`) will
 automatically call the proxy's `/shutdown` endpoint when it completes.
 
-In the future, Kubernetes will hopefully support more container lifecycle hooks
-that will allow Linkerd to handle these situations automatically.
+For security reasons, the proxy's `/shutdown` endpoint is disabled by default.
+In order to be able to manually shutdown the proxy, you must enable this
+endpoint by installing Linkerd with the `--set enableShutdownEndpoint=true`
```
Comment (alpeb) [inline linkerd.io/content/2.16/tasks/graceful-shutdown.md:163]: Likewise, should be `proxy.enableShutdownEndpoint=true`

**Review by alpeb — APPROVED** [review]
LGTM modulo my two comments above.

**Pull request merged**